tempesta_fw/http_msg.c

```diff
@@ -945,14 +971,21 @@ __tfw_http_msg_alloc(int type, bool full)
 						      TFW_POOL_ZERO)
 			 : (TfwHttpMsg *)tfw_pool_new(TfwHttpResp,
 						      TFW_POOL_ZERO);
-	if (!hm)
+	if (!hm) {
+		TFW_WARN("Insufficient memory to create %s message\n",
+			 ((type & Conn_Clnt) ? "request" : "response"));
 		return NULL;
+	}
+
+	BUILD_BUG_ON(FIELD_SIZEOF(TfwHttpMsg, flags) * BITS_PER_BYTE
+		     < _TFW_HTTP_FLAGS_NUM);
 
 	if (full) {
 		hm->h_tbl = (TfwHttpHdrTbl *)tfw_pool_alloc(hm->pool,
 							    TFW_HHTBL_SZ(1));
 		if (unlikely(!hm->h_tbl)) {
-			TFW_WARN("Insufficient memory to create message\n");
+			TFW_WARN("Insufficient memory to create %s message\n",
+				 ((type & Conn_Clnt) ? "request" : "response"));
```

Comment:
Not  big deal, but usually it helpful to have different messages: having one in dmesg you can quickly understang which code exactly produced the message. Also, as it's in your code, good to have as much of the message in the same line for easier `grep`'ing the code.